src/main/kotlin/net/lumalyte/lg/infrastructure/persistence/guilds/GuildRepositorySQLite.kt

```
@@ -148,6 +149,26 @@ class GuildRepositorySQLite(private val storage: Storage<Database>) : GuildRepos
 
+    private fun migrateAllyHomeColumns() {
+        val columns = listOf(
+            "ally_home_world TEXT",
+            "ally_home_x INTEGER",
+            "ally_home_y INTEGER",
+            "ally_home_z INTEGER"
+        )
+        for (colDef in columns) {
+            try {
+                storage.connection.executeUpdate("ALTER TABLE guilds ADD COLUMN $colDef")
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)